sphinx/source/tutorial/exercises/stocks.py

```diff
@@ -21,7 +21,15 @@
 output_file("stocks.html", title="stocks.py example")
 
 # create a figure
-p1 = figure(title="Stocks")
+p1 = figure(
+    title="Stocks",
+    x_axis_label="Date",
+    y_axis_label="Close price",
+    x_axis_type="datetime")
```

Comment:
First, thanks for the contribution!!!

Can I suggest you something like this to save a line?

``` python
p1 = figure(title="Stocks", 
            x_axis_label="Date",
            y_axis_label="Close price",
            x_axis_type="datetime")
```
and the same with the `dict` below.
This is a minor and we can take care of this later, but if you can do it now, it would be great!
Thanks!!